### DISCUSSION TITLE
Temporary bug prioritises boundaries over place nodes

### DISCUSSION THREAD

**Discussion by danieldegroot2** [osm-search/Nominatim#3152] [General]
I just had a very temporary bug on openstreetmap.org where searching i.e. "Amsterdam" or "Amsterdam, NL" (capital of european mainland The Netherlands) would return the judicial boundary instead of the place node/boundary, and those would be the only two main results. I tried also with a different city (returned district boundary instead of place node/boundary) and on a different zoom level and moving around and this wall still the case. I then tried to reproduce it with nominatim.openstreetmap.org, but failed to do so. Going back to openstreetmap.org and trying again it seemed to have resolved itself.

Odd bug. Just thought I'd let you know.

**Comment by lonvia** [comment]
This can happen during updates where there is a very small window of opportunity to get different results while a place is being updated. It's difficult to avoid.
